packages/core/admin/admin/src/components/Webhooks/HeadersInput/index.js

```diff
@@ -1,143 +1,107 @@
-                  name={`${name}.${index}.key`}
-                  options={options}
-                  styles={customStyles(get(errors, `headers.${index}.key`, null))}
-                  value={formatOption(key)}
-                />
-              </section>
-              <section>
-                <InputText
-                  className={get(errors, `headers.${index}.value`, null) && 'bordered'}
-                  onChange={onChange}
-                  name={`${name}.${index}.value`}
-                  value={value}
-                />
-              </section>
-              <div>
-                <CircleButton type="button" isRemoveButton onClick={() => onRemove(index)} />
-              </div>
-            </li>
-          );
+    <Stack size={1}>
+      <FieldLabel>
+        {formatMessage({
+          id: 'Settings.webhooks.form.headers',
+          defaultMessage: 'Headers',
         })}
-      </ul>
-      <button onClick={() => onClick(name)} type="button">
-        <Plus fill="#007eff" width="10px" />
-        <FormattedMessage
-          id="Settings.webhooks.create.header"
-          defaultMessage="Create a new header"
+      </FieldLabel>
+      <Box padding={8} background="neutral100" hasRadius>
+        <FieldArray
+          validateOnChange={false}
+          name="headers"
+          render={({ push, remove }) => (
+            <Grid gap={4}>
+              <GridItem col={6}>
+                <FieldLabel>
+                  {formatMessage({
+                    id: 'Settings.webhooks.key',
+                    defaultMessage: 'Key',
+                  })}
+                </FieldLabel>
+              </GridItem>
+              <GridItem col={6}>
+                <FieldLabel>
+                  {formatMessage({
+                    id: 'Settings.webhooks.value',
+                    defaultMessage: 'Value',
+                  })}
+                </FieldLabel>
+              </GridItem>
+              {values.headers?.map((header, i) => (
+                // eslint-disable-next-line
+                <React.Fragment key={i}>
+                  <GridItem col={6}>
+                    <Field
+                      as={TextInput}
+                      name={`headers.${i}.key`}
+                      aria-label={`row ${i + 1} key`}
+                      error={
+                        errors.headers?.[i]?.key && formatMessage({ id: errors.headers[i]?.key })
+                      }
+                    />
+                  </GridItem>
+                  <GridItem col={6}>
+                    <Row>
+                      <Box style={{ flex: 1 }}>
+                        <Field
+                          as={TextInput}
+                          aria-label={`row ${i + 1} value`}
+                          name={`headers.${i}.value`}
+                          error={
+                            errors.headers?.[i]?.value &&
+                            formatMessage({ id: errors.headers[i]?.value })
```

Comment:
Same here, the purpose is to have less noise in our tests.

Reply:
All good. WDYT about importing the whole language to avoid any noise ?
```
import en from '../../../../translations/en.json';

const makeApp = component => {
  const history = createMemoryHistory();
  const messages = { en };
  const localeNames = { en: 'English' };

  return (
    <LanguageProvider messages={messages} localeNames={localeNames}>
      <Theme>
        <Router history={history}>{component}</Router>
      </Theme>
    </LanguageProvider>
  );
};
```